libs/sdk-core/src/models.rs

```diff
@@ -640,6 +641,7 @@ pub struct PaymentExternalInfo {
 #[derive(Default)]
 pub struct ListPaymentsRequest {
     pub filters: Option<Vec<PaymentTypeFilter>>,
+    pub metadata_filters: Option<String>,
```

Comment:
I'm not sure if filter or filters is better here. I lean towards metadata_filter because its not a list.
```suggestion
    pub metadata_filter: Option<String>,
```